assets/css/style.css

```diff
@@ -119,6 +122,210 @@ header {
   height: 150%;
 }
 
+.product-categories {
+  display: flex;
+  column-gap: 8px;
+  padding: 0;
+}
+
+.type_dev {
+  display: flex;
+  column-gap: 8px;
+  padding: 0;
+}
+
+.article_text {
+  width: 95%;
+  margin: 0 auto;
+}
+
+.article_img {
+  width: 95%;
+  margin: 0 auto;
+}
+
+.button-category {
+  padding: 10px 15px;
+  border: none;
+  border-radius: 10px;
+  color: #6070ff;
+  background-color: #ebebff;
+}
+
+.li-color {
+  color: #344563;
+}
+
+.div-works {
+  /* display: flex;
+  flex-direction: column; */
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+
+  /* border: 3px solid; */
+  border-radius: 20px;
+  padding: 10px 10px 18px 10px;
+  box-shadow: 0 1px 3px 1px #dfe1e6;
+  background-color: white;
+}
+
+.button-project,
+.button-get-resume {
+  border: 1px solid #396df2;
+  border-radius: 10px;
+  padding: 15px 10px;
+  font-size: 17px;
+  background: none;
+  color: #396df2;
+  font-weight: 500;
+  width: fit-content;
+}
+
+.works-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  row-gap: 100px;
+  background-color: #fbf8f8;
+  margin-bottom: 40px;
+}
```

Comment:
- [ ] From the project requirement this `.works-section` which serves as the container for the whole works card should be styled using the css grid layour while each card should use the display flex. please kindly adhere to the project requirement as shown below and make the required changes :+1: 
- project requirements
![req](https://github.com/kambereBr/Portfolio/assets/89055502/b54e1462-b308-4df2-a62a-5131a0f0f12e)